cluster/charts/universal-crossplane/templates/bootstrapper/versions-configmap.yaml

```diff
@@ -7,5 +7,4 @@ metadata:
     {{- include "labelsBootstrapper" . | nindent 4 }}
 data:
   crossplaneVersion: {{ (trimPrefix "v" .Values.image.tag) }}
-  xgqlVersion: {{ (trimPrefix "v" .Values.xgql.image.tag) }}
```

Comment:
I'm not sure whether this one should also be wrapped with the condition, it looks like it's used by the build submodule https://github.com/upbound/build/blob/master/makelib/controlplane.mk#L22

Reply:
Good catch!
No, we still need this. It is mainly used by Upbound Cloud to get the installed versions.
This should not necessarily be inside the bootstrapper directory since it is for UXP, keeping it there should be ok for now.